scrooge-generator/src/main/resources/scalagen/finagleService.scala

```diff
@@ -119,6 +120,7 @@ class {{ServiceName}}$FinagleService(
   // ---- end boilerplate.
 
 {{/hasParent}}
+  private[this] val scopedStats = if (serviceName != "") stats.scope(serviceName) else stats
```

Comment:
ok, can you open a PR on finagle-thrift too then, so that we can thread the name through properly with the rich client interface?